src/cli.ts

```diff
@@ -303,6 +303,7 @@ async function open(options: Options, url: string | undefined, enableRecorder: b
 
   switch (language) {
     case 'javascript': 
+    case undefined:
```

Comment:
The case of `undefined` is a bit unclear from the code style point of view in my mind. Would be nicer if we could reuse the `enableRecorder` parameter for it, since its false for opening a website and true if we want to use the recorder.

Reply:
How about now?